.github/workflows/release.yml

```diff
@@ -182,6 +182,14 @@ jobs:
   build-test-docker:
     uses: ./.github/workflows/build-test-docker.yaml
     secrets: inherit
+    with:
```

Comment:
Oh ok I think I understand now.
So maybe you can add in the build-test-docker.yaml toplevel comment something like
"This is mostly independent of semgrep. The goal is to factorize xxx?
This is mostly a small wrapper around docker/setup-buildx-action@v2 and docker/metadata-action@v4".

Even though you mention hardcoded path like scripts/validate-docker in it ...